engines/csvengine.py

```diff
@@ -39,7 +39,7 @@ class engine(Engine):
         "double": "REAL",
         "decimal": "REAL",
         "char": "TEXT",
-        "bool": "INTEGER",
+        "bool": "REAL",
```

Comment:
Seems like this should be `INTEGER` for the same reason as SQLite since in concept it should be 0 or 1. Is there a reason for making it `REAL`?